socorro/unittest/config/jenkins.py.dist

```diff
@@ -25,7 +25,7 @@ oldDatabaseName.default = 'socorro_test'
 
 databaseUserName = cm.Option()
 databaseUserName.doc = 'the user name for the database servers'
-databaseUserName.default = 'breakpad_rw'
+databaseUserName.default = 'test'
```

Comment:
Same as for commonconfig -- this should get deprecated eventually in favor of configman style config. Until then, I recommend using the 'test' user as that's what we use in our unit test configs.